UserInterface/ViewModels/SettingsViewModel.cs

```diff
@@ -0,0 +1,121 @@
+﻿using BusinessLogic.Interfaces;
+using BusinessLogic.Services;
+using System;
+using System.ComponentModel;
+using System.Windows;
+using System.Windows.Input;
+using System.Windows.Media;
+using UserInterface.Commands;
+using UserInterface.Stores;
+using Xceed.Wpf.Toolkit;
+using static System.Windows.Forms.VisualStyles.VisualStyleElement.TrayNotify;
+using BusinessLogic.Classes;
+using BusinessLogic.IRepositories;
+using DataAccess.Repositories;
+using BusinessLogic.Models;
+using UserInterface.ViewModels.Modals;
+
+namespace UserInterface.ViewModels
+{
+    internal class SettingsViewModel : ViewModelBase, INotifyPropertyChanged
+    {
+        #region Dependencies
+        private readonly NavigationStore navigationStore;
+        private readonly SettingService settingService;
+
+        #endregion Dependencies 
+
+        #region Commands
+        public ICommand SaveSetting => new Command(ValidateColor);
+
+        #endregion Commands
+
+        #region Properties
+        private SolidColorBrush savedColor;
+        public SolidColorBrush SavedColor
+        {
+            get
+            {
+                return savedColor;
+            }
+            set
+            {
+                savedColor = value;
+                OnPropertyChanged(nameof(SavedColor));
+            }
+        }
+
+        private System.Windows.Media.Color selectedColor;
+        public  System.Windows.Media.Color SelectedColor
+        {
+            get
+            {
+                return selectedColor;
+            }
+            set
+            {
+                selectedColor = value;
+                OnPropertyChanged(nameof(SelectedColor));
+            }
+        }
+
+        protected BusinessLogic.Classes.Color convertedColor;
+        private Settings savedSettings { get; set; }
+        #endregion Properties
+
+        #region Constructor
+
+        public SettingsViewModel(NavigationStore navigationStore)
+        {
+            this.navigationStore = navigationStore;
+            this.navigationStore.AddPreviousViewModel(new EmployeePortalViewModel(navigationStore));
+            this.navigationStore.HideTopBar = false;
+
+            ISettingsRepository settingsRepository = new SettingsRepository();
+            settingService = new SettingService(settingsRepository);
+            GetSetting();
+        }
+        #endregion Constructor
+
+        private void GetSetting()
```

Comment:
Misschien kijken of je van deze functie 1 kan maken? je gebruikt hem hier en in MainWindow ?